napari/viewer.py

```diff
@@ -190,3 +190,13 @@ def current_viewer() -> Optional[Viewer]:
         return _QtMainWindow.current_viewer()
     except ImportError:
         return None
+
+
+def all_open_viewers() -> list:
+    """Return a list of all open napari viewers, or None."""
+    try:
+        from napari._qt.qt_main_window import _QtMainWindow
+
+        return _QtMainWindow.all_open_viewers()
+    except ImportError:
```

Comment:
in what situation would we run into an `ImportError`?

Reply:
If you expand the existing code slightly above this, you will see that this is the same pattern used by `current_viewer`

```python
def current_viewer() -> Optional[Viewer]:
    """Return the currently active napari viewer."""
    try:
        from napari._qt.qt_main_window import _QtMainWindow
        return _QtMainWindow.current_viewer()
    except ImportError:
        return None
```

I don't know the reason for why that was important the first time. Possibly @alisterburt might remember, but this was years ago.

Reply:
I've removed the try/except import pattern from both `napari/viewer.py` `current_viewer()` and `all_open_viewers()`.